src/Microsoft.AspNetCore.Razor.Language/Legacy/TokenizerBackedParser.cs

```diff
@@ -21,14 +23,28 @@ protected TokenizerBackedParser(LanguageCharacteristics<TTokenizer> language, Pa
 
             var languageTokenizer = Language.CreateTokenizer(Context.Source);
             _tokenizer = new TokenizerView<TTokenizer>(languageTokenizer);
-            Span = new SpanBuilder(CurrentLocation);
```

Comment:
Q: Why did CurrentLocation go away everywhere? I don't actually think it's used any more either. Could nuke

Reply:
Because we no longer need SourceLocations when building the Green tree. The diagnostics use `CurrentStart`. So, `CurrentLocation` is unused.